### DISCUSSION TITLE
Bar labels font size

### DISCUSSION THREAD

**Discussion by Thommenek** [nmalkin/plot-likert#31] [Q&A] [answered]
Hello!
This package is marvelous! However, I'm wondering how I can change the bar labels' font size?
Does anyone know?

Regards,
Tom

**Comment by nmalkin** [comment, marked as answer]
Here's a solution that's a bit hacky, but it should work:

```python
axes = plot_likert.plot_likert(data, plot_likert.scales.agree, plot_percentage=True,
                               bar_labels=True, bar_labels_color="snow", colors=plot_likert.colors.default_with_darker_neutral);

from matplotlib.text import Annotation

for child in axes.get_children():
    if type(child) == Annotation:
        child.set_fontsize('xx-large')
```

The font size [can be a number or any of the following values](https://matplotlib.org/stable/api/text_api.html#matplotlib.text.Text.set_fontsize): `'xx-small', 'x-small', 'small', 'medium', 'large', 'x-large', 'xx-large'`

The result looks like this:
<img width="619"  src="https://user-images.githubusercontent.com/903179/171559199-8e55c6e2-e90d-4993-8a57-99b81b39dd66.png">
(You can then adjust or remove the individual [Annotation](https://matplotlib.org/stable/api/text_api.html#matplotlib.text.Annotation)s if they don't fit, etc.)

**Reply by Thommenek** [reply]
Works perfectly!
Thank you very much!

Kind regards,
Tom

**Reply by Thommenek** [reply]
One addition that I discovered seconds ago: instead xx-large, x-large, etc., you can just write an integer: [...] child.set_fontsize('15").
Using a number one can have stronger control of the size of labels.

Have a nice day!
